tensorflow_lite_support/python/task/processor/proto/nl_classification_options_pb2.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NL Classification options protobuf."""
+
+import dataclasses
+from typing import Any, Optional
+
+from tensorflow_lite_support.cc.task.processor.proto import nl_classification_options_pb2
+from tensorflow_lite_support.python.task.core.optional_dependencies import doc_controls
+
+_NLClassificationOptionsProto = nl_classification_options_pb2.NLClassificationOptions
+
+
+@dataclasses.dataclass
+class NLClassificationOptions:
+  """Configure the input/output tensors for NL classification processor.
+
+  - No special configuration is needed if the model has only one input tensor
+  and one output tensor.
+
+  - When the model has multiple input or output tensors, use the following
+  configurations to specifiy the desired tensors:
+    -- tensor names: `input_tensor_name`, `output_score_tensor_name`,
+    `output_label_tensor_name`
+    -- tensor indices: `input_tensor_index`, `output_score_tensor_index`,
+    `output_label_tensor_index`
+  Tensor names has higher priorities than tensor indices in locating the
+  tensors. It means the tensors will be first located according to tensor
+  names. If not found, then the tensors will be located according to tensor
+  indices.
+
+  - Failing to match the input text tensor or output score tensor with
+  neither tensor names nor tensor indices will trigger a runtime error.
+  However, failing to locate the output label tensor will not trigger an
+  error because the label tensor is optional.
+
+  Attributes:
+    input_tensor_name: Name of the input text tensor, if the model has
+      multiple inputs. Only the input tensor specified will be used for
+      inference; other input tensors will be ignored. Defaults to "INPUT".
+    output_score_tensor_name: Name of the output score tensor, if the model has
+      multiple outputs. Defaults to "OUTPUT_SCORE".
+    output_label_tensor_name: Name of the output label tensor, if the model has
+      multiple outputs. Defaults to "OUTPUT_LABEL". By default, label file
+      should be packed with the output score tensor through Model Metadata. See
+      the MetadataWriter for NLClassifier [1]. NLClassifier reads and parses
```

Comment:
docstring will be parsed as markdown so you can link to the MetadataWriter for NLClassifier site using Markdown syntax.